eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -248,18 +253,18 @@ impl Eth2NearRelay {
                     {
                         Ok(()) => {
                             info!(target: "relay", "Successful light client update submission!");
-                            self.current_gap_between_finalized_and_signature_slot = 96;
+                            self.current_gap_between_finalized_and_signature_slot = 97;
```

Comment:
Use constant for this (see my previous comment on this too)

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/823867b1ba6f883c2ba735cd2f94c2910f26b069